next-app/next.config.js

```diff
@@ -0,0 +1,10 @@
+/** @type {import('next').NextConfig} */
+
+const nextConfig = {
+    reactStrictMode: true,
+    compiler: {
+        styledComponents: true,
```

Comment:
if you're using styled components in a next app you should ahve this in your config

Reply:
it also accepts an object with lots of optoins instead of `true` if you want more control